Source/JavaScriptCore/heap/MachineStackMarker.cpp

```diff
@@ -539,7 +539,7 @@ void* MachineThreads::Thread::Registers::stackPointer() const
 #error Unknown Architecture
 #endif
 
-#elif defined(__GLIBC__) && ENABLE(JIT)
+#elif OS(LINUX) && ENABLE(JIT)
```

Comment:
@zdobersek musl intentionally does not define __MUSL__ as they want to remain posix compliant nothing more. this conditional controls the ucontext behavior which is similar between glibc and musl as well as uclibc, so this covers all the libcs on linux.  One OS that may not be covered now is kfreebsd since thats not linux and uses glibc. If that matters, I can add __FreeBSD_kernel__ && __GLIBC__ to the check along.